src/App.js

```diff
@@ -26,33 +27,123 @@ const generateSquares = () => {
 }
 
 const App = () => {
-
+  
   const [squares, setSquares] = useState(generateSquares());
+  const [turn, setTurn] = useState(true);
+  const [winnerExist, setWinner] = useState(false);
+  const [winner,showWinner] = useState("Let's play!");
 
-  // Wave 2
-  // You will need to create a method to change the square 
-  //   When it is clicked on.
-  //   Then pass it into the squares as a callback
 
+  const changeSquare = (changedSquare)=>{
```

Comment:
The way the program is written, if a play accidentally clicks a space that has already been claimed, they lose their turn. A check could be added to this function to address that.